index.js

```diff
@@ -65,3 +66,7 @@ function whyIsNodeRunning (logger) {
     }
   }
 }
+
+function normalizeFileName(fileName) {
+  return fileName.startsWith('file://') ? fileURLToPath(fileName) : fileName
```

Comment:
In ESM mode paths for the file name can start with the `file://` protocol, so we need to strip that out to get the same output as before.